.coveragerc

```diff
@@ -0,0 +1,8 @@
+[report]
+exclude_lines =
+    pragma: no cover
+    @abc.abstractmethod
+<<<<<<< HEAD
+    pos = init_pos
+=======
+>>>>>>> 092ad40... Add pytest-cov config file
```

Comment:
Here is still a merge conflict 😄

Reply:
Whoops, can you check if there are still instances of this in other files?